style.css

```diff
@@ -301,11 +299,10 @@ hr {
   border: none;
 }
 
-@media only screen and (max-width: 400px) {
+@media only screen and (max-width: 375px) {
   body {
-    width: 100%;
+    width: 375px;
     margin: 1px 2px;
-    overflow: scroll;
   }
```

Comment:
- Kindly remove media for the mobile version. This is used in the desktop version and used as min-width. This gives problems to your code.